vm/src/lib.rs

```diff
@@ -68,6 +68,7 @@ mod frozen;
 pub mod function;
 pub mod import;
 pub mod obj;
+mod py_io;
```

Comment:
py_io, py_serde but pyobject, pystr.
would there be a standard for the name?

Reply:
Yeah, I noticed that too lol. I think `py_io`/`py_serde` make sense because it's a "py" version of io/serde, but then a "pyobject" is it's own sort of thing. Not too sure about pystr

Reply:
That make sense, but It seems not obvious so it would make confusion again in future.